services/activity/.env.test

```
@@ -4,6 +4,4 @@ PORT="3006"
 DATABASE_URL="postgresql://admin:password@localhost:5432/vers"
 API_IDENTIFIER="http://localhost:3000/"
 REPLAY_SERVICE_URL="http://localhost:3009"
-SERVICE_AUTH_PRIVATE_KEY="-----BEGIN PRIVATE KEY-----
-MC4CAQAwBQYDK2VwBCIEIBMom57erggdVdDCIdRWS+NKMykK+I5BUKpuHziAq+0W
------END PRIVATE KEY-----"
\ No newline at end of file
+SERVICE_AUTH_PRIVATE_KEY="-----BEGIN PRIVATE KEY-----\nMC4CAQAwBQYDK2VwBCIEIDIZ7D9KuJWlGEhRDOr5dQHy5WKSOK0K+3nELix9LwN5\n-----END PRIVATE KEY-----\n"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)